### ISSUE TITLE
Create Stream Default Value

### ISSUE THREAD

**Issue by lpf4254302** [lf-edge/ekuiper#2293]
**What would you like to be added/modified**:
now:
create stream my_stream (id bigint, name string, score float)
->
create stream my_stream (id bigint, name string, score float **default 0.0**)

**Why is this needed**:
Business need

**Comment by Elia-Renzoni** [comment]
Hi @ngjaying 
I'd like to work on this issue if is still relevant for the project. I just have a couple of questions: 
1) Should the default value be defined directly in each field declaration, as shown in the example?
2) And is this expected to work for all data types? For example: ``` create stream my_stream (id bigint, name string default "foo", score float default 1.2) ```

Thanks!

**Comment by ngjaying** [comment]
@Elia-Renzoni Yes, this is still a valid feature request. Thanks for taking this.
